srcpkgs/nextcloud-client/template

```diff
@@ -59,13 +59,13 @@ nextcloud-client-kwallet_package() {
 	pkg_install() {
 		vbin ${FILESDIR}/kwallet/nextcloud.kwallet
 		vmkdir usr/share/applications
-		cp build/src/gui/com.nextcloud.desktopclient.nextcloud.desktop \
-		   ${DESTDIR}/usr/share/applications/nextcloud-kwallet.desktop
+		vcopy build/src/gui/com.nextcloud.desktopclient.nextcloud.desktop \
+		   usr/share/applications/nextcloud-kwallet.desktop
 		vsed -i -e 's,^Exec=.*,Exec=/usr/bin/nextcloud.kwallet,' \
 		     -e 's,\(^Name=.*\),\1 - use kwallet,' \
 		     -e 's,\(^Comment=.*\),\1 - use kwallet credential storage,' \
 		     -e '/^# Translations/,$d' \
-		     ${DESTDIR}/usr/share/applications/nextcloud-kwallet.desktop
+		   ${PKGDESTDIR}/usr/share/applications/nextcloud-kwallet.desktop
```

Comment:
please move this logic to do_install and just vmove the files here in pkg_install

Reply:
And it should be `vinstall` not `vcopy`.

Reply:
> And it should be `vinstall` not `vcopy`.

It cannot be a vinstall as the new nextcloud-kwallet.desktop file doesn't exist until this point, because the kwallet is a void-made wrapper not something from upsteam (see #26774). The do_install should be a little neater for this, but let me know if you'd like another solution.

Reply:
It can!